libraries/AP_Generator/AP_Generator_Cortex.h

```diff
@@ -0,0 +1,127 @@
+/*
+ * This file is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This file is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Author: Oliver Walters / Currawong Engineering Pty Ltd
+ */
+
+#pragma once
+
+#include "AP_Generator_config.h"
+#include <AP_PiccoloCAN/AP_PiccoloCAN.h>
+
+#if HAL_GENERATOR_ENABLED && AP_GENERATOR_CORTEX_ENABLED
+
+#include "AP_Generator_Backend.h"
+#include <AP_Common/AP_Common.h>
+#include <AP_Logger/AP_Logger_config.h>
+#include <AP_PiccoloCAN/AP_PiccoloCAN_Device.h>
+#include <AP_PiccoloCAN/piccolo_protocol/CortexPackets.h>
+#include <stdint.h>
+#include <stdio.h>
```

Comment:
```suggestion
```

shouldn't need stdio.

We omit stdint even 'though it's probably strictly required.